CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 59.2.2
```

Comment:
This should be a minor change (i.e. 59.3.0) as it's adding a new method. See: https://semver.org/